frontend/src/Components/SubmitProgress.js

```diff
@@ -3,7 +3,7 @@ import React from 'react'
  const SubmitProgress = (props) => {
      const {onClick,pendingApiCall,text,disabled} = props;
   return (
-        <button className='btn btn-primary'onClick={onClick} disabled={disabled}>
+        <button className='btn btn-primary mt-2'onClick={onClick} disabled={disabled}>
```

Comment:
In terms of readability and ease of development, it would be better to set the ui library change a little earlier.